component/src/main/java/me/androidbox/component/agenda/VisitorFilter.kt

```diff
@@ -0,0 +1,102 @@
+package me.androidbox.component.agenda
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import me.androidbox.component.R
+import me.androidbox.component.general.TaskButton
+import me.androidbox.component.ui.theme.*
+import java.util.*
+
+@Composable
+fun VisitorFilter(
+    modifier: Modifier = Modifier,
+    listOfVisitorType: List<VisitorType>,
```

Comment:
This isn't really necessary. Since it's an enum, you could just
VisitorType.values() and loop it directly. So no need to receive it as parameter

Reply:
@MKiperszmid Removed this as a parameter